packages/internal/whenable.js

```diff
@@ -0,0 +1,19 @@
+/* global globalThis */
```

Comment:
AFAICT, this module is never imported by anyone. I tried deleting it locally and nothing seems to break. What is it supposed to be for. Should you just delete it?

Reply:
It seems the `powers` exported here are meant to be an argument to `prepareWhenableModule`. I notice that above you said

> To preserve the correct layering, I'd put a `prepareWhenableModule` wrapper in `@agoric/internal`, have it close over the `DisconnectionObject`.

which seems relevant ;)

Reply:
Yeah, that's as much as I could disentangle without creating circular references between `@agoric/internal` and `@agoric/whenable`.

Reply:
Seems untangled. Resolving.